test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -2655,12 +2598,12 @@
       "path": "microsoft.azure.functions.javaworker/2.0.0",
       "hashPath": "microsoft.azure.functions.javaworker.2.0.0.nupkg.sha512"
     },
-    "Microsoft.Azure.Functions.NodeJsWorker/4.0.0": {
+    "Microsoft.Azure.Functions.NodeJsWorker/3.0.0": {
```

Comment:
Does this node worker version correlate to v4?

Reply:
... this may be out of date now -- we'll have to regen this file i think